### PR TITLE
Boundingbox demo - located in /demo/boundingbox/

### PR DESCRIPTION
**Pull request by alx**

### CODE REVIEW

**Comment by alx** [comment]
It should be working, I need to request the api to test.

Ugly labels at the moment, style should come tomorrow.

File: demo/boundingbox/dist/index.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  <div class="boundingBox" data-service="face_gender" data-image-list="http://i.imgur.com/QiksDIs.jpg,https://www.nbc.com/sites/nbcunbc/files/files/styles/1080xauto/public/scet/photos/269/7924/PAA_267.JPG,http://wiinoob.walyou.netdna-cdn.com/wp-content/uploads/2009/03/mii-cupcakes.jpg"></div>
+  <hr/>
+  <div class="boundingBox" data-service="clothes"></div>
+  <hr/>
+  <div class="boundingBox" data-service="generic"></div>
+
+</div>
+
+  <script>__REACT_DEVTOOLS_GLOBAL_HOOK__ = parent.__REACT_DEVTOOLS_GLOBAL_HOOK__</script>
+  <script type="text/javascript" src="/app.js"></script>
```
Comment (beniz) [inline demo/boundingbox/dist/index.html]: In my testing setup I had to replace with `src="assets/app.js`, is this a bug or a wrong setup from my part ?

File: demo/boundingbox/src/index.html [hunk 2]
```diff
@@ -13,7 +13,7 @@
 
   <div class="boundingBox" data-service="faces" data-image-list="http://i.imgur.com/QiksDIs.jpg,https://www.nbc.com/sites/nbcunbc/files/files/styles/1080xauto/public/scet/photos/269/7924/PAA_267.JPG,http://wiinoob.walyou.netdna-cdn.com/wp-content/uploads/2009/03/mii-cupcakes.jpg"></div>
   <hr/>
-  <div class="boundingBox" data-service="face_gender" data-image-list="http://i.imgur.com/QiksDIs.jpg,https://www.nbc.com/sites/nbcunbc/files/files/styles/1080xauto/public/scet/photos/269/7924/PAA_267.JPG,http://wiinoob.walyou.netdna-cdn.com/wp-content/uploads/2009/03/mii-cupcakes.jpg"></div>
+  <div class="boundingBox" data-service="face_gender" data-image-list="http://i.imgur.com/QiksDIs.jpg,https://www.nbc.com/sites/nbcunbc/files/files/styles/1080xauto/public/scet/photos/269/7924/PAA_267.JPG,http://wiinoob.walyou.netdna-cdn.com/wp-content/uploads/2009/03/mii-cupcakes.jpg" data-confidence="0.5"></div>
```
Comment (beniz) [inline demo/boundingbox/src/index.html]: Just a detail, would be great to keep the parameter's name from API, `confidence_threshold`, possibly changed as `confidence-threshold` if needed.

**Comment by beniz** [comment]
Thanks :) I guess `app.js` needs to be rebuilt, but I can't locate the instructions. Maybe they could be added to `dist/README.md` or even better to a `README.md` at the root of `demo/boundingbox` ?

File: demo/boundingbox/package.json [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    "release:patch": "npm version patch && npm publish && git push --follow-tags",
+    "serve": "node server.js --env=dev",
+    "serve:dist": "node server.js --env=dist",
+    "start": "node server.js --env=dev",
+    "test": "karma start",
+    "test:watch": "karma start --autoWatch=true --singleRun=false"
+  },
+  "repository": "",
+  "keywords": [],
+  "author": "Your name here",
```
Comment (beniz) [inline demo/boundingbox/package.json]: Put your name here :)

**Comment by beniz** [comment]
Here is what I had to do to build the `app.js`:
```
sudo apt-get install npm
sudo npm install copyfiles -g
sudo npm install webpack -g
sudo npm install bower-webpack-plugin
sudo npm install babel
sudo npm install babel-core
sudo npm install babel-loader
sudo npm install eslint
sudo npm install eslint-loader
sudo npm install eslint-plugin-react
sudo npm install babel-eslint
```
then
```
npm run copy
webpack --env=dist
```
but I'm getting
```
ERROR in ./src/index.js
Module build failed: ReferenceError: Unknown plugin "transform-decorators-legacy" specified in "/home/beniz/projects/deepdetect/dev/alx/deepdetect/demo/boundingbox/.babelrc" at 0, attempted to resolve relative to "/home/beniz/projects/deepdetect/dev/alx/deepdetect/demo/boundingbox"
    at /home/beniz/projects/deepdetect/dev/alx/deepdetect/demo/boundingbox/node_modules/babel-core/lib/transformation/file/options/option-manager.js:180:17
    at Array.map (native)
    at Function.normalisePlugins (/home/beniz/projects/deepdetect/dev/alx/deepdetect/demo/boundingbox/node_modules/babel-core/lib/transformation/file/options/option-manager.js:158:20)
    at OptionManager.mergeOptions (/home/beniz/projects/deepdetect/dev/alx/deepdetect/demo/boundingbox/node_modules/babel-core/lib/transformation/file/options/option-manager.js:234:36)
    at OptionManager.init (/home/beniz/projects/deepdetect/dev/alx/deepdetect/demo/boundingbox/node_modules/babel-core/lib/transformation/file/options/option-manager.js:368:12)
    at File.initOptions (/home/beniz/projects/deepdetect/dev/alx/deepdetect/demo/boundingbox/node_modules/babel-core/lib/transformation/file/index.js:212:65)
    at new File (/home/beniz/projects/deepdetect/dev/alx/deepdetect/demo/boundingbox/node_modules/babel-core/lib/transformation/file/index.js:135:24)
    at Pipeline.transform (/home/beniz/projects/deepdetect/dev/alx/deepdetect/demo/boundingbox/node_modules/babel-core/lib/transformation/pipeline.js:46:16)
    at transpile (/home/beniz/projects/deepdetect/dev/alx/deepdetect/demo/boundingbox/node_modules/babel-loader/lib/index.js:46:20)
    at Object.module.exports (/home/beniz/projects/deepdetect/dev/alx/deepdetect/demo/boundingbox/node_modules/babel-loader/lib/index.js:155:20)
```

**Comment by alx** [comment]
README.md added.

Missing dependency in your list of lib installed with npm.

It's better to use the simple ```npm install``` or ```yarn install``` to install all dependencies when a package.json file is available.

**Comment by beniz** [comment]
`yarn install` says:

```
ERROR: [Errno 2] No such file or directory: 'install'
```

**Pull request closed without merging**